allensdk/brain_observatory/behavior/behavior_session.py

```diff
@@ -34,6 +30,11 @@
 from allensdk.internal.api import db_connection_creator
 
 
+class BehaviorBase:
+    # TODO remove this
```

Comment:
Does this still need to be here?

Reply:
`BehaviorSession` is under construction, will be removed in next PR